lib/state_manager/adapters/active_record.rb

```diff
@@ -67,11 +67,11 @@ def self.included(base)
 
             # In the AR use case, we don't want to run any callbacks
             # until the model has been saved
-            def run_before_callbacks(*args)
+            def run_before_callbacks(*args, **kwargs)
               self.pending_transition = args
             end
 
-            def run_after_callbacks(*args)
+            def run_after_callbacks(*args, **kwargs)
             end
```

Comment:
non-blocking comment:
Ruby 2.7 let's you do `(...)` instead (argument forwarding syntax)